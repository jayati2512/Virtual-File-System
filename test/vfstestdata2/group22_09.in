unmountvfs demo09
createvfs demo09 25
mountvfs demo09
listdir /home/dir1/ 0 /tmp/listdir1
createvfs demo09 20
makedir /home/ dir1
addfile /home/dir1/ file1 /usr/include/zlibdefs.h
listfile /home/dir1/file1 /tmp/listfile1
makedir /home/dir2/ dir3
makedir / bin
addfile /home/dir2/ file2.txt /usr/include/zlibdefs.h
copyfile /home/ /bin/
searchfile file /tmp/searchfile1
searchfile file /tmp/searchfile1
unmountvfs demo09
