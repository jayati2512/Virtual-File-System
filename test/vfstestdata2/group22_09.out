VFS_NOT_MOUNTED
createvfs_SUCCESS
mountvfs_SUCCESS
CANNOT_FIND_SPECIFIED_PATH_OR_DIR
DATA_FILE_ALREADY_EXISTS
makedir_SUCCESS
addfile_SUCCESS
NOT_A_TEXT_FILE
makedir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS
copyfile_SUCCESS
searchfile_SUCCESS
searchfile_SUCCESS
unmountvfs_SUCCESS
